docs/source/install.rst

```diff
@@ -123,3 +123,35 @@ channel:
 .. _`Arch Linux` : https://www.archlinux.org/
 .. _`Hitchhiker's Guide to Python` : http://docs.python-guide.org/en/latest/
 .. _`Homebrew` : http://brew.sh
+
+Installing in ``conda`` environments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+It is recommended to install Zipline in an isolated ``conda`` environment. 
+Installing Zipline in ``conda`` environment will not interfere your default 
+Python deployment or site-packages, which will prevent any possible conflict 
+with your global libraries. For more information on ``conda environment``, see 
+the `Conda User Guide <https://conda.io/docs/user-guide/tasks/manage-environments.html>`_
+
+Assuming ``conda`` has been set up, you can create a virtual environment:
```

Comment:
Let's call this a "conda env", just to disambiguate it from a "virtualenv" virtual environment.